admin/services/jwt.service.js

```diff
@@ -4,6 +4,7 @@ const moment = require('moment')
 const crypto = Promise.promisifyAll(require('crypto'))
 const jwt = Promise.promisifyAll(require('jsonwebtoken'))
 const uuidv4 = require('uuid/v4')
+const config = require('../config')
```

Comment:
Linting error: 'config' is assigned a value but never used.

Reply:
This should now be fixed as i have re-enabled codacy and standardJS should enforce this by default.